internal/engine/wazevo/ssa/instructions.go

```diff
@@ -648,6 +648,10 @@ const (
 	// `fence`.
 	OpcodeFence
 
+	// OpcodeMemoryWait ...
+	// `v = memory_wait MemFlags, t, x, p`.
+	OpcodeMemoryWait
```

Comment:
Got pointer that I should be able to define host func call without a separate opcore will check it out